journal-of-systematic-palaeontology.csl

```diff
@@ -171,6 +183,7 @@
       <key macro="author-count" names-min="3" names-use-first="3"/>
       <key macro="author" names-min="3" names-use-first="1"/>
       <key macro="year-date"/>
+     <key macro="month-date"/>
```

Comment:
I actually disagree with this as a reading of the styleguide: it says nothing about what level "chronological" refers to. I think it makes little sense (and looks completely random) to sort by parts of the date that aren't actually displayed anywhere, which is why I typically use year-date in sort macros. 
If there's any indication that the style actually means "chronological" quite literally, we should just use <key variable="issued"/>

I'd be inclined to close this